### PR TITLE
[12.0][MIG] base_delivery_carrier_label: Migration to 12.0

### PR DESCRIPTION
**Pull request by bealdav**
replace #205

### CODE REVIEW

**Comment by bealdav** [comment]
mergeable

**Comment by rousseldenis** [comment]
@bealdav extreme measures :smile:  in a hurry ?

**Review by rousseldenis — APPROVED** [review]

**Comment by rousseldenis** [comment]
@bealdav Could you maybe squash last commit in Migration one ?

**Comment by bealdav** [comment]
c bien parce que c toi

**Comment by rousseldenis** [comment]
I try to stay coherent beyond OCA repos :innocent:

**Comment by bealdav** [comment]
@rousseldenis Thanks for your involvement in all these repos ;-)

**Comment by yvaucher** [comment]
@rousseldenis @bealdav thanks. I'm glad this is merged.

File: base_delivery_carrier_label/models/stock_picking.py [hunk 1]
```diff
@@ -85,23 +74,36 @@ def get_shipping_label_values(self, label):
...
+    def generate_labels(self):
+        """ Legacy method. Remove me after 12.0
+        """
+        _logger.warning(
+            "Your delivery module depending on your carrier must call "
+            "action_generate_carrier_label() method "
+            "instead of generate_labels()")
+        return self.action_generate_carrier_label()
+
+    def _set_a_default_package(self):
```
Comment (rousseldenis) [inline base_delivery_carrier_label/models/stock_picking.py:86]: @bealdav Could you give me a little context to have a package mandatory in this case ? Thanks

**Pull request merged**